website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -1,24 +1,21 @@
 ## How to use this component
 
 Super Select is a custom select-like component aiming to overcome some limitations of the HTML `<select>` element.
-It is based on the popular Ember [PowerSelect](https://ember-power-select.com/) add-on.
 
-We provide two main components with similar APIs, `SuperSelect::Single` and `SuperSelect::Multiple`.
+It's a light wrapper for Ember [PowerSelect](https://ember-power-select.com/), with some specific accessibility and styling choices made to best fit our design system.
+
+We provide two main components with similar APIs: `SuperSelect::Single` and `SuperSelect::Multiple`.
 
 **There are two ways to use each of the Super Select component types:**
 
-1. `Form::SuperSelect::Single::Base` or `Form::SuperSelect::Multiple::Base`—the base component with just the Super Select control.
+1. `Form::SuperSelect::Single::Base` or `Form::SuperSelect::Multiple::Base`—the base component with just the Super Select control. In these instances, you will need to add your own label, helper text, and error messaging.
 2. `Form::SuperSelect::Single::Field` or `Form::SuperSelect::Multiple::Field`—the field parent component which includes the Super Select control, label, helper text, and error messaging (in a wrapping container).
 
-We recommend using the Field component as it provides built-in accessibility functionality. Use the Base component for custom layouts or special use cases not covered by the Field component.
-
-### Field components
-
-#### Basic invocations including search
+We recommend using the Field variation as it provides built-in accessibility functionality. Try to only use the Base variation for custom layouts or special use cases not otherwise covered.
```

Comment:
Adding "try" here weakens the recommendation I think. We already tell them that we recommend the Field version as it has built-in accessibility. So if we say to "try" to use the Base version only for custom layouts etc. that doesn't sound like as strong of a recommendation to me.